applications/crossbar/doc/faxes.md

```diff
@@ -226,6 +226,17 @@ curl -v -X GET \
     http://{SERVER}:8000/v2/accounts/{ACCOUNT_ID}/faxes/outbox/{FAX_ID}
 ```
 
+#### Resubmit a fax from the outbox folder
+
+> POST /v2/accounts/{ACCOUNT_ID}/faxes/outbox/{FAX_ID}
+
+```curl
+curl -v -X POST \
```

Comment:
@lazedo hey can you double check what we did for conference actions?  I believe that we used PUT... and if so should here as well for consistency